static/app/components/numberInput.tsx

```diff
@@ -0,0 +1,51 @@
+import {forwardRef, ForwardRefRenderFunction, useRef} from 'react';
+import {useLocale} from '@react-aria/i18n';
+import {AriaNumberFieldProps, useNumberField} from '@react-aria/numberfield';
+import {useNumberFieldState} from '@react-stately/numberfield';
+
+import Input, {InputStylesProps} from 'sentry/components/input';
+import mergeRefs from 'sentry/utils/mergeRefs';
+
+interface NumberInputProps
```

Comment:
shall we export the `Props`?

Reply:
done! https://github.com/getsentry/sentry/pull/41549/commits/7da439cacf555ca683fc85af333c9d6bb41d7884